webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js

```diff
@@ -1,4 +1,12 @@
+import URI from 'urijs';
 import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const inventoryUrl = path =>
   foremanUrl(`/foreman_inventory_upload/${path}`);
+
+export const getInventoryDocsUrl = () =>
+  foremanUrl(
+    `/links/manual/+?root_url=${URI.encode(
```

Comment:
In foreman 24 there is no links controller yet, we have to return the external URL directly